keyboards/planck/keymaps/Huguesmmm/keymap.c

```diff
@@ -0,0 +1,288 @@
+ * |  Del |   1  |   2  |   3  |   =  |   &  |   *  |  (   |   )  |  {   |  }   | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   #  |   4  |   5  |   6  |   +  |   -  |   _  |  <   |   >  |  [   |  ]   |  ¨   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |   7  |   8  |   9  |  0   |   /  |   \  |  ~   |      |      | Ins  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      | Prev | Vol- | Vol+ | Play |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_MACLOWER] = LAYOUT_planck_grid(
+    KC_DEL,  KC_1,    KC_2,    KC_3,    CA_EQL, CA_AMPR, CA_ASTR, CA_LPRN, CA_RPRN, CA_LCBR, CA_RCBR, KC_BSPC,
+    KC_GRV,  KC_4,    KC_5,    KC_6,    CA_PLUS, CA_MINS, CA_UNDS, CA_LABK,   CA_RABK,   CA_LBRC, CA_RBRC, CA_DIAE,
+    _______, KC_7,    KC_8,    KC_9,    KC_0,    CA_SLSH,  CA_BSLS, CA_DTIL, _______, _______, KC_INS, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, KC_MPRV, KC_VOLD, KC_VOLU, KC_MPLY
+),
+
+    /* Keyshortcut
+ * ,-----------------------------------------------------------------------------------------.
+ * |cmd+ctrl+esc|      |      |      |      |      |      |      |      |      |      |      |
+ * |------------+------+------+------+------+------+------+------+------+------+------+------|
+ * |            |      |      |      |      |      |      |      |      | Lock |      |      |
+ * |------------+------+------+------+------+------+------+------+------+------+------+------|
+ * |            |      |      |      |      |      |      |      |      |      |      |      |
+ * |------------+------+------+------+------+------+------+------+------+------+------+------|
+ * |            |      |      |      |      |      |      |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------------'
+ */
+[_MACKEYSHORTCUT] = LAYOUT_planck_grid(
+    KC_COE, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, MACOS_LCK, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+),
+};
+
+#ifdef AUDIO_ENABLE
+  float plover_song[][2]     = SONG(PLOVER_SOUND);
+  float plover_gb_song[][2]  = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+      state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+      state = update_tri_layer_state(state, _MACLOWER, _MACRAISE, _ADJUST2);
```

Comment:
Something tells me this is the only value ever to be returned? The core [tri layer feature](https://docs.qmk.fm/#/feature_tri_layer) should be used instead.